script.js

```diff
@@ -0,0 +1,25 @@
+var block = document.getElementById("test");
+var left = 0;
+var top1 = 0;
+var bottom = 0;
+var right = 0;
+
+document.onkeydown = function (event){
+  console.log(event);
+  if (event.key == "ArrowRight"){
```

Comment:
I could be wrong, but you should use keyCode instead of key 
learn how to use keyCode and replace all keys here